lib/lunchbot_web/templates/admin/user/index.html.heex

```diff
@@ -21,8 +21,17 @@
             <%= filter_string_input(:user, :role, @conn.params) %>
           </div>
         
-        
-        
+          <div class="field">
+            <label>Hashed Password</label>
+            <%= filter_select(:user, :hashed_password, @conn.params) %>
```

Comment:
missed this one

Reply:
oh I see, this is just for filtering?

Reply:
> oh I see, this is just for filtering?

Yep! That's just for finding users on the right hand side